internal/mode/static/state/graph/httproute.go

```diff
@@ -760,6 +762,44 @@ func validateFilterRedirect(
 	return allErrs
 }
 
+func validateFilterRewrite(
+	validator validation.HTTPFieldsValidator,
+	filter v1.HTTPRouteFilter,
+	filterPath *field.Path,
+) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if filter.URLRewrite == nil {
+		panicForBrokenWebhookAssumption(errors.New("urlRewrite cannot be nil"))
```

Comment:
can we also add a case for it here https://github.com/nginxinc/nginx-gateway-fabric/blob/c493dc9551094d87a89a08ebbfa7df080c108b0c/internal/mode/static/state/change_processor_test.go#L2000 ?